tests/test_pythoncapi_compat_cext.c

```diff
@@ -2448,6 +2448,46 @@ test_tuple(PyObject *Py_UNUSED(module), PyObject *Py_UNUSED(args))
     return test_tuple_fromarray();
 }
 
+// Test adapted from CPython's _testcapi/object.c
+static int MyObject_dealloc_called = 0;
+
+static void
+MyObject_dealloc(PyObject *op)
```

Comment:
Would you mind to rename it to `TryIncref_dealloc()` and rename also `MyObject_dealloc_called`?